### ISSUE TITLE
Enable opsgenie incident.io integration for devops alerts

### ISSUE THREAD

**Issue by danieldides** [sourcegraph/sourcegraph#30762]
> Going forward each time you receive an alert in Ops Genie a new Incident should be created. This includes even the known issues.
> For example, if a service is having high load and we already have a github ticket in to fix it, we should still create a new INC until this is resolved. This will allow others to see that we are getting too many of these alerts that need to be worked on ASAP.

via https://sourcegraph.slack.com/archives/C02Q9G9A59S/p1644263709165809

Enable the [OpsGenie](https://docs.incident.io/opsgenie) incident.io integration to automatically create these alerts.

**Comment by daxmc99** [comment]
Blocked on current Opsgenie plan (as of Feb 1st)

**Comment by danieldides** [comment]
Further elaboration: our current Opsgenie plan tier does not support any sort of API requests so we cannot directly link OpsGenie to something like Incident.io. To enable this feature would cost a good bit more per month.

One alternative is to look into [Grafana OnCall](https://grafana.com/products/oncall/) to replace OpsGenie, which we may already be paying for indirectly on our Grafana Cloud plan.

**Comment by kalanchan** [comment]
tldr - either pay for OpsGenie feature, or switch alerting services.
